app/templates/errors/error404.html

```diff
@@ -0,0 +1,24 @@
+﻿{% extends "base.html" %}
+{% block style %}
+<link rel="stylesheet" href="../../static/content/errorstyle.css">
```

Comment:
```suggestion
<link rel="stylesheet" href="{{ url_for('static',filename='content/errorstyle.css' }}">
```
Following [doc here for `url_for()`](https://flask.palletsprojects.com/en/1.1.x/quickstart/#static-files) and [guide here for `Blueprint`](https://flask.palletsprojects.com/en/1.1.x/blueprints/#static-files). If moving the static and template files to `/errors` folder for the `Blueprint`, then `'static'` should be replaced with `'.static'`. I'm aware that most of the templates haven't made this change yet, and we will look to changing all of them to this convention.